.github/release-drafter.yml

```
@@ -1,3 +1,3 @@
 # https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.adoc
 _extends: .github
-tag-template: support-core-$NEXT_MINOR_VERSION
\ No newline at end of file
+tag-template: support-core-$NEXT_MINOR_VERSION 
```

Not sure about that space, I'd rather remove it :)

```suggestion
tag-template: support-core-$NEXT_MINOR_VERSION
```

---

Done!